cmd/storj-sim/network.go

```diff
@@ -301,3 +317,15 @@ func identitySetup(network *Processes) (*Processes, error) {
 
 	return processes, nil
 }
+
+func vipReadString(into *string, dir, flagName string) error {
```

Comment:
vipReadString isn't a very clear name.  Not that viperReadString is much better.

Reply:
It's because we need the setting that is store in the config file for a particular entry.

For example let's say person runs `storj-sim network setup` then changes the gateway address. Now `storj-sim network test` needs to expose that server address in `GATEWAY_0_ADDR`... so this means we need to read that from the config file with that hack.

Reply:
I'm still missing something.  Doesn't storj-sim pick up on the config file changes @
https://github.com/storj/storj/blob/331f0cc8457d1156199653b58761330c7eaab7fa/cmd/storj-sim/process.go#L42 ??

why can't we just...
storj-sim --host "::1" network setup
# overwrite gateway server.address
storj-sim network test

Reply:
Because we need to expose the gateway/0 `server.address` as an environment variable `GATEWAY_0_ADDR` to the [test script](https://github.com/storj/storj/blob/master/scripts/test-sim-aws.sh#L39), which is invoked with `storj-sim network test bash ./test-sim-aws.sh`.

Reply:
ooooooooh.  thanks for being patient

Reply:
also name fixed